lib/ember_cli/runner.rb

```diff
@@ -32,9 +35,11 @@ def run!(command)
 
     private
 
-    def write(output, streams:)
-      streams.each do |stream|
-        stream.write(output)
+    def redirect(stream, pipes:)
+      Tee.open(*pipes, stdout: nil) do |pipe|
+        while line = stream.gets
```

Comment:
Is this the best way to redirect `STDERR` / `STDOUT` to multiple output streams? The use case is to redirect `STDOUT` to both a log file and `STDOUT` (which explains the dependence on a `tee` Ruby implementation, given [the trouble we've had with using a shell `tee` implementation](https://github.com/thoughtbot/ember-cli-rails/issues/417#issuecomment-197991094))

@thoughtbot/shell I'm unsure if this `while` loop (within a Ruby block) will block other output streams (imagine a case where a command is writing to `STDOUT`, encounters an error, wants to write to `STDERR`, then continue to write to `STDOUT`)